core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/listeners/YouTubePlayerListener.kt

```diff
@@ -14,7 +15,7 @@ interface YouTubePlayerListener {
      * Called every time the state of the player changes. Check [PlayerConstants.PlayerState] to see all the possible states.
      * @param state a state from [PlayerConstants.PlayerState]
      */
-    fun onStateChange(youTubePlayer: YouTubePlayer, state: PlayerConstants.PlayerState)
+    fun onStateChange(youTubePlayer: YouTubePlayer, state: PlayerConstants.PlayerState, view: View? = null)
```

Comment:
I find this confusing, why do we need to add a nullable view?

This is problematic for a few reasons:
1. Is a breaking change. Every user of the library will have to change their implementation of `onStateChange`
2. The API is not clear

Maybe this should be a separate method instead?

Reply:
Changes are made as per suggestion:

1. Revert code for `onStateChange` method definition, removed `view` parameter
2. Create separate method for handle callback of `onShowCustomView` and `onHideCustomView`